app/models/services/service_credential_binding_view.rb

```diff
@@ -10,7 +10,7 @@ module Types
         Sequel.as(:service_keys__id, :id),
         Sequel.as(:service_keys__guid, :guid),
         Sequel.as(Types::SERVICE_KEY, :type),
-        Sequel.as(:spaces__guid, :space_guid),
+        Sequel.as(:spaces__id, :space_id),
```

Comment:
I think you can change the alias to `Sequel.as(:service_instances__space_id, :space_id)` and remove [this JOIN](https://github.com/cloudfoundry/cloud_controller_ng/blob/ecbc74972ad27cf1e0055d6760ab35cb185ff170/app/models/services/service_credential_binding_view.rb#L40).